common/src/main/java/com/distkv/common/NodeInfo.java

```diff
@@ -26,6 +29,8 @@ private NodeInfo(Builder builder) {
     this.address = builder.getAddress();
     this.nodeId = builder.getNodeId();
     this.isMaster = builder.isMaster();
+    this.status = NodeStatus.RUNNING;
```

Comment:
Is it reasonable to make it RUNNING by default?

Reply:
I didn't think about it. I'll add a todo